inspirehep/modules/workflows/utils.py

```diff
@@ -176,3 +177,41 @@ def download_file_to_workflow(workflow, name, url):
         if req.status_code == 200:
             workflow.files[name] = req.raw
             return workflow.files[name]
+
+
+def store_original_record_for_source(record_id, source, data):
```

Comment:
The API (this function and the next) sounds a bit awkward: names are long, but do not explain themselves very well. That said, good names come from their users, and since this PR does not include uses it's hard to tell which better names we could give them.